packages/api/cms-api/src/dam/files/entities/folder.entity.ts

```diff
@@ -82,7 +82,7 @@ export function createFolderEntity({ Scope }: { Scope?: Type<DamScopeInterface>
         @Field()
         archived: boolean;
 
-        @Property({ columnType: "boolean" })
+        @Property({ columnType: "boolean", default: false })
```

Comment:
Shouldn't this be a separate PR?

Reply:
this change is not from me, maybe this chang is applied because @thomasdax98 changed the target branch to ubernext. I will update my PR to ubernext